docs/sources/mimir/configure/about-versioning.md

```diff
@@ -159,6 +161,7 @@ The following features are currently experimental:
   - `-distributor.limit-inflight-requests-using-grpc-method-limiter`
   - `-ingester.limit-inflight-requests-using-grpc-method-limiter`
 - Logging of requests that did not send any HTTP request: `-server.http-log-closed-connections-without-response-enabled`.
+- Allow gRPC status codes to be used in `status_code` label of `cortex_request_duration_seconds` metric: `-server.report-grpc-codes-in-instrumentation-label-enabled`.
```

Comment:
This is not marked as experimental, right?

Reply:
You are right, I will fix this.